### PR TITLE
Add Prydwen Institute presence

### PR DESCRIPTION
**Pull request by kyrie25**
* Home page
image
* Other sites
image
image
image
image

### CODE REVIEW

File: websites/P/Prydwen Institute/dist/metadata.json [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+  "contributors": [
+    {
+      "name": "Sharkie",
+      "id": "400618023075184650"
+    },
+    {
+      "name": "Antillar",
+      "id": "277496099315515392"
+    }
+  ],
```
Comment (EncryptedDev) [inline websites/P/Prydwen Institute/dist/metadata.json:16]: Have they contributed at least 25% to the codebase? 🤔 
image

**Review by EncryptedDev — CHANGES_REQUESTED** [review]
Check my review

File: websites/P/Prydwen Institute/dist/metadata.json (hunk 1 above)
Comment (kyrie25) [reply websites/P/Prydwen Institute/dist/metadata.json:16]: Well the both of them provided ALL the Art Assets for the presence and one of them is the site owner whom made the final decision on what to show up on `largeImage` and `details`
If that counts as contributing to every `Image` and `details` line then I guess so

Comment (EncryptedDev) [reply websites/P/Prydwen Institute/dist/metadata.json:16]: As the presence author you should own the art assets on your Discord Developer account. Furthermore, neither of those reasons would qualify for presence contributor

Comment (kyrie25) [reply websites/P/Prydwen Institute/dist/metadata.json:16]: Got it. Will omit ASAP.

**Review by Slowlife01 — APPROVED** [review]

**Review by EncryptedDev — APPROVED** [review]

**Pull request merged**